code/firmware/rosco_m68k_firmware/InterfaceReference.md

```diff
@@ -44,6 +44,12 @@ as possible, and will be kept updated as firmware 2.0 is developed.
       * 1.2.2.5 PRINTCHAR (Function #4)
       * 1.2.2.6 SETCURSOR (Function #5)
       * 1.2.2.9 CHECKCHAR (Function #6)
+      * 1.2.2.8 CHECK_DEVICE_SUPPORT (Function #7)
+      * 1.2.2.9 GET_DEVICE_COUNT (Function #8)
+      * 1.2.2.10 GET_DEVICE (Function #9)
+      * 1.2.2.11 DEVICE_RECVCHAR (Function #10)
+      * 1.2.2.12 DEVICE_SENDCHAR (Function #11)
+      * 1.2.2.13 DEVICE_CHECKCHAR (Function #12)
```

Comment:
It looks like there's a slight mismatch between this listing and the headings below:
```suggestion
      * 1.2.2.7 CHECKCHAR (Function #6)
      * 1.2.2.8 CHECK_DEVICE_SUPPORT (Function #7)
      * 1.2.2.9 GET_DEVICE_COUNT (Function #8)
      * 1.2.2.10 GET_DEVICE (Function #9)
      * 1.2.2.11 ADD_DEVICE (Function #10)
      * 1.2.2.12 DEVICE_RECVCHAR (Function #11)
      * 1.2.2.13 DEVICE_SENDCHAR (Function #12)
      * 1.2.2.14 DEVICE_CHECKCHAR (Function #13)
      * 1.2.2.15 RESERVED (Function #14)
      * 1.2.2.16 RESERVED (Function #15)
      * 1.2.2.17 DEVICE_CTRL (Function #16)
```